lib/util/makeRequest.ts

```diff
@@ -70,34 +83,47 @@ export type Options = request.CoreOptions &
  */
 export async function makeRequest(
   options: Options,
-  reportError: jsonParser.ReportError,
+  reportError: jsonParser.ReportError
 ): Promise<SwaggerObject> {
-  const promise = new Promise<SwaggerObject>((resolve, reject) => {
-    request(options, (err, response, responseBody) => {
+  return retry(
+    async (bail, retryNr) => {
+      const { err, response, responseBody } = await promisifiedRequest(options)
+
       if (err) {
```

Comment:
err is going to be if the request fails ? we need to handle also error return codes